OpenEdXMobile/res/values/styles.xml

```diff
@@ -45,6 +45,10 @@
         </item>
     </style>
 
+    <style name="NoDisplayTheme" parent="android:style/Theme.NoDisplay">
```

Comment:
style name should be 'AppTheme.' as per the naming convention we are following in the project.